recipe/build.sh

```diff
@@ -3,7 +3,8 @@
 cp $BUILD_PREFIX/share/gnuconfig/config.* .
 
 export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
-export CFLAGS="-I${PREFIX}/include  ${CFLAGS}"
+# workaround #71 sqlite 3.36 issue
```

Comment:
For our future selves when we want to know why we had to do this :-)
Item 3 in https://www.sqlite.org/releaselog/3_36_0.html

    An error is raised on any attempt to access the rowid of a VIEW or subquery. Formerly, the rowid of
    a VIEW would be indeterminate and often would be NULL. The -DSQLITE_ALLOW_ROWID_IN_VIEW
    compile-time option is available to restore the legacy behavior for applications that need it.